index.html

```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Concert in Bolivia</title>
+  <link rel="stylesheet" href="styles.css">
+  <link rel="stylesheet" href="//fonts.googleapis.com/earlyaccess/nanumgothic.css">
+</head>
+
+<body>
```

Comment:
- [ ] Kindly for all headers in the project make the underline width smaller to match the Behance design.

| Your design | Behance design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/169203361-00d387ad-3cb4-46af-a6aa-cad35fd93f90.png) | ![image](https://user-images.githubusercontent.com/24703171/169203406-2aef9ece-51e7-4d91-8091-a2f6fd41b67e.png) |

Reply:
ok, the issue is remmediated.